stonesoup/sensor/radar/radar.py

```diff
@@ -489,9 +491,10 @@ def measure(self, sky_state, noise=True, **kwargs):
         det_prob = self.gen_probability(sky_state)[0]
         # Is the state detected?
         if np.random.rand() <= det_prob:
-            self.measurement_model.translation_offset = self.translation_offset
-            self.measurement_model.rotation_offset = self.rotation_offset
+            measurement_model = copy.deepcopy(self.measurement_model)
+            measurement_model.translation_offset = self.translation_offset
+            measurement_model.rotation_offset = self.rotation_offset
```

Comment:
Will you not have the same issue here with the two offsets being pointers to properties which will evolve over time?

Reply:
A good point...